feature/gribi/ate_tests/ipv4_entry_test/ipv4_entry_test.go

```diff
@@ -57,36 +57,48 @@ var (
 	dutPort1 = attrs.Attributes{
 		Desc:    "DUT Port 1",
 		IPv4:    "192.0.2.1",
+		IPv6:    "2001:db8::192:0:2:1",
 		IPv4Len: 30,
+		IPv6Len: 126,
 	}
 	dutPort2 = attrs.Attributes{
 		Desc:    "DUT Port 2",
 		IPv4:    "192.0.2.5",
+		IPv6:    "2001:db8::192:0:2:5",
 		IPv4Len: 30,
+		IPv6Len: 126,
 	}
 	dutPort3 = attrs.Attributes{
 		Desc:    "DUT Port 3",
 		IPv4:    "192.0.2.9",
+		IPv6:    "2001:db8::192:0:2:9",
 		IPv4Len: 30,
+		IPv6Len: 126,
 	}
 
 	atePort1 = attrs.Attributes{
 		Name:    "atePort1",
 		Desc:    "ATE Port 1",
 		IPv4:    "192.0.2.2",
+		IPv6:    "2001:db8::192:0:2:2",
 		IPv4Len: 30,
+		IPv6Len: 126,
 	}
 	atePort2 = attrs.Attributes{
 		Name:    "atePort2",
 		Desc:    "ATE Port 2",
 		IPv4:    "192.0.2.6",
+		IPv6:    "2001:db8::192:0:2:6",
 		IPv4Len: 30,
+		IPv6Len: 126,
 	}
 	atePort3 = attrs.Attributes{
 		Name:    "atePort3",
 		Desc:    "ATE Port 3",
 		IPv4:    "192.0.2.10",
+		IPv6:    "2001:db8::192:0:2:a",
 		IPv4Len: 30,
+		IPv6Len: 126,
 	}
 )
```

Comment:
What happens if we just remove the IPv6 addresses here and instead just enable IPv6 under the interface? Will that pass the test?

Reply:
I have removed the configuring ipv6 address and added enabling ipv6 in the interface